src/main.rs

```diff
@@ -123,6 +123,29 @@ fn main() {
                         .long("password")
                         .env("HYDRA_PASSWORD")
                         .help("A user password"),
+                )
+                .after_help(
+                    "Here is an example JSON config:
+
+    {
```

Comment:
Would it be possible to serialize and pretty print a `JobsetConfig` instead? This could avoid future divergences between the struct and the example.

Reply:
Yeah definitely! :+1: